samples/net/mqtt_azure/README.rst

```diff
@@ -0,0 +1,50 @@
+.. _mqtt-azure-sample:
+
+MQTT Azure Sample
+#################
+
+Overview
+********
+
+This sample application demonstrates how a MQTT client
+can publish messges to Azure Cloud IoT hub based on MQTT protocol.
+
+- Acquire a DHCPv4 lease
+- Establish a TLS connection with Azure Cloud IoT hub
+- Publish data to the Azure cloud
+- SOCKS5 supported
+
+The source code of this sample application can be found at:
+:zephyr_file:`samples/net/mqtt-azure`.
+
+Requirements
+************
+
+- Azure Cloud account
+- Azure IOT Cloud credentials and required information
+- Freedom Board (FRDM-K64F)
+- Network connectivity
+
+Building and Running
+********************
+This application has been built and tested on the NXP FRDMK64F.
+Certs/keys are required to authenticate to the Azure Cloud IoT hub.
+
+Users will also be required to configure the following Kconfig options
+based on their Azure Cloud IoT Hub:
+
+- SAMPLE_CLOUD_AZURE_USERNAME - Username field use
+         {iothubhostname}/{device_id}/?api-version=2018-06-30,
+         where {iothubhostname} is the full CName of the IoT hub.
+- SAMPLE_CLOUD_AZURE_PASSWORD    - Password field, use a SAS token.
+- SAMPLE_CLOUD_AZURE_CLIENT_ID   - ClientId field, use the deviceId.
+- SAMPLE_CLOUD_AZURE_HOSTNAME    - IoT hub hostname
+- SAMPLE_CLOUD_AZURE_SERVER_ADDR - IP address of the Azure MQTT broker
+- SAMPLE_CLOUD_AZURE_SERVER_PORT - Port number of the Azure MQTT broker
+
+In case if you are running application behind proxy, set below details
+- SAMPLE_SOCKS_ADDR - IP address of SOCKS5 Proxy server
+- SAMPLE_SOCKS_PORT - Port number of SOCKS5 Proxy server
+
```

Comment:
I was expecting to see the commands to build and run this sample, and some sample output.